docs/tutorials/prowler-app-sso.md

```diff
@@ -0,0 +1,89 @@
+# Configuring SAML Single Sign-On (SSO) in Prowler
+
+This guide explains how to enable and test SAML SSO integration in Prowler. It includes environment setup, certificate configuration, API endpoints, and how to configure Okta as your Identity Provider (IdP).
+
+---
+
+## Environment Configuration
+
+### `DJANGO_ALLOWED_HOSTS`
+
+Update this variable to specify which domains Django should accept incoming requests from. This typically includes:
+
+- `localhost` for local development
+- container hostnames (e.g. `prowler-api`)
+- public-facing domains or tunnels (e.g. ngrok)
+
+**Example**:
+
+```env
+DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1,prowler-api,mycompany.prowler
```

Comment:
Does this work if the value is `*`?

Reply:
It seems like it’s possible, although not recommended

Reply:
I mean if SAML will work having `DJANGO_ALLOWED_HOSTS=*` because that's the only way as of today to configure it if running behind an ALB with dynamic IPs.

Reply:
But this isn’t something new introduced by SSO, it already existed and will have to remain as-is in the Cloud environment. Nothing has changed in that regard